CHANGELOG.md

```diff
@@ -2,6 +2,45 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
+## 2.0.1 - TBD
```

Comment:
For the 2.0.0 readiness, the 2.0.1 listing should not be present; add it after tagging.